src/Domain/DoctrineDonationRepository.php

```diff
@@ -85,6 +85,38 @@ public function findWithExpiredMatching(\DateTimeImmutable $now): array
         return array_map(static fn(array $row): UuidInterface => $row['uuid'], $rows);
     }
 
+    /**
+     * @return Donation[]
+     */
+    public function findWithBigGiveWgmf25Matching(Campaign $campaign): array
```

Comment:
the implementation here doesn't seem to have anything specific to WGMF 25, so slightly confusing name, but not worth checking since it looks like this is code to run once and then delete.